app/src/components/integrations/elements/integrationSettings/integrationForm/integrationForm.scss

```diff
@@ -79,4 +92,20 @@
   button {
     font-size: 13px;
   }
+
+  &:last-child {
+    button {
+      color: $COLOR--topaz-2;
+
+      &:hover {
+        color: $COLOR--topaz-hover-2;
+      }
+      &:focus {
+        color: $COLOR--topaz-focused;
+      }
+      &:active {
+        color: $COLOR--topaz-pressed;
+      }
+    }
+  }
```

Comment:
Why are these changes not done in the `Button` component?

Reply:
Usually the components from Design system are not subject to override, especially if the Design systems developed by us.
So please update colors inside of the `Button` component from `componentLibrary`.

Reply:
Done.